bin.js

```diff
@@ -171,7 +171,8 @@ const result = commist()
       ],
       boolean: [
         'help',
-        'private'
+        'private',
+        'noOpen'
```

Comment:
This option can be named `open`, then users can do `--no-open` and minimist handles it automatically. Pass a `default: { open: true }` option to retain the open-by-default behaviour.

Reply:
![Neat!](https://i.imgur.com/Y4YFthE.gif)

Reply:
@goto-bus-stop updated as recommended, cheers 👍